### PR TITLE
Change master branch to main in everywhere

### PR DESCRIPTION
**Pull request by soilSpoon**

### CODE REVIEW

**Comment by soilSpoon** [comment]
Sorry, I didn't consider the other parts in #212, so there were 2 PRs.

File: examples/counter/src/App.js [hunk 1]
```diff
@@ -85,6 +85,26 @@ const Controls = () => {
...
+
+const ButtonDown = ({ onClick }) => (
+  <svg viewBox="0 0 430 452" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      onClick={onClick}
+      d="M214.83 451.523C347.216 451.523 429.602 360.898 429.602 226.523C429.602 187.816 422.852 152.786 410.112 122.5L106 426.214C136.689 442.604 173.299 451.523 214.83 451.523Z"
+      fill="#FFBEC2"
+    />
+  </svg>
+)
```
Comment (dai-shi) [inline examples/counter/src/App.js:107]: Oh, you want to put them in. Okay, why not.

**Review by dai-shi — APPROVED** [review]
Thanks again for the fix. We would need to to the same in jotai repo.

File: examples/counter/src/App.js (hunk 1 above)
Comment (soilSpoon) [reply examples/counter/src/App.js:107]: > Oh, you want to put them in. Okay, why not.

I'm sorry, I'm not good at English, can you explain in more detail?

Comment (dai-shi) [reply examples/counter/src/App.js:107]: It was intentional to avoid having `ButtonUp` and `ButtonDown` there. But, we can have them.

Comment (soilSpoon) [reply examples/counter/src/App.js:107]: I added it using ButtonUp and ButtonDown in lines 83~84.

Is this the problem now?

Comment (dai-shi) [reply examples/counter/src/App.js:107]: No, it's not a problem. I will merge this.
(As there was an intention not to have it, we may remove it in the future.)

**Pull request merged**